test/EndToEnd/ValidateInsertedManifests.cs

```diff
@@ -31,10 +31,14 @@ public void ManifestReaderCanReadManifests()
 
                     string manifestFile = manifestDir.GetFile("WorkloadManifest.json").FullName;
 
-                    File.Exists(manifestFile).Should().BeTrue();
-                    using var fileStream = new FileStream(manifestFile, FileMode.Open, FileAccess.Read);
-                    Action readManifest = () => WorkloadManifestReader.ReadWorkloadManifest(manifestId, fileStream, manifestFile);
-                    readManifest.ShouldNotThrow("manifestId:" + manifestId + " manifestFile:" + manifestFile + "is invalid");
+                    var baselineFile = manifestDir.GetFile("Baseline.WorkloadSet.json").FullName;
+                    if (!(new FileInfo(baselineFile).Exists))
```

Comment:
I was confused by this at first, now I see that this is preventing it from trying to validate that the workload set is a valid manifest.  I think it would probably be clearer to skip the logic if `manifestId` is `workloadsets` rather than checking for the baseline workload set file.

Reply:
It seems I was wrong about the root of why it started failing after I made this adjustment: this was totally valid, but the 'manifestId' variable didn't correspond to the manifest ID...whoops...

Fixed that, and now it passes tests.